lib/compilers/argument-parsers.ts

```diff
@@ -1088,12 +1059,12 @@ export class ZigCxxParser extends ClangParser {
         return ['c++', '--help'];
     }
 
-    static override getHiddenHelpOptions(exampleFile: string): string[] {
-        return ['c++', '-mllvm', '--help-list-hidden', exampleFile, '-S', '-o', '/tmp/output.s'];
+    static override getHiddenHelpOptions(): string[] {
+        return ['c++', '-mllvm', '--help-list-hidden', '-x', 'c++', '/dev/null', '-S', '-o', './output.s'];
     }
 
-    static override getStdVersHelpOptions(exampleFile: string): string[] {
-        return ['c++', '-std=c++9999999', exampleFile, '-S', '-o', '/tmp/output.s'];
+    static override getStdVersHelpOptions(): string[] {
+        return ['c++', '-std=c++9999999', '-x', 'c++', '/dev/null', '-S', '-o', './output.s'];
```

Comment:
why did you change to `./output.s`? you won't have write access to `./` at all

Reply:
ah! like you said in discord, only if isolate=true which is probably is not. I'll revert this (we need to fix golang then)